### PR TITLE
docs: onboarding users

### PR DESCRIPTION
**Pull request by hifabienne**
Description on how to open the registration form directly in the login UI and about the registration options form.

### Definition of Ready

- [ ] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [x] PR is linked to the corresponding user story
- [x] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/guides/integrate/onboarding/end-users.mdx [hunk 1]
```diff
@@ -22,6 +22,34 @@ If you want to start automating the process of onboarding your users and let the
...
+You can test the impact of the different prompts on your login UI in our [OIDC Playground](/docs/apis/openidoauth/authrequest).
+
+Unfortunately, SAML doesn't offer the same level of control over the initial screen.
+You won't be able to directly influence which page (login or registration) is shown through the SAML flow.
+
+#### Choose Registration Option
+
+If an organization does allow local user registration as well as registration with a social or enterprise login, the user will have to choose the option.
+
+As soon as the user clicks the "register" button, the registration options screen will be shown.
```
Comment (mffap) [inline docs/docs/guides/integrate/onboarding/end-users.mdx:41]:
```suggestion
As soon as users click the "register" button, they will be presented with a screen showing the different registration options.
```

File: docs/docs/guides/integrate/onboarding/end-users.mdx [hunk 2]
```diff
@@ -22,6 +22,34 @@ If you want to start automating the process of onboarding your users and let the
...
+
+<img src="/docs/img/guides/solution-scenarios/register-options.png" alt="Register Options"/>
+
+The use can now select either local user registration or an external provider.
```
Comment (mffap) [inline docs/docs/guides/integrate/onboarding/end-users.mdx:45]:
```suggestion
After that, the user can select either local user registration or an external provider.
```

File: docs/docs/guides/integrate/onboarding/end-users.mdx [hunk 3]
```diff
@@ -22,6 +22,34 @@ If you want to start automating the process of onboarding your users and let the
...
+By pressing the button of an external provider, the user will directly be redirected to the provider.
```
Comment (mffap) [inline docs/docs/guides/integrate/onboarding/end-users.mdx:46]:
```suggestion
By pressing the button of an external provider, the user will directly be redirected to the provider for consent.
```

**Review by mffap — CHANGES_REQUESTED** [review]
Some smaller suggetions and typos.

File: docs/docs/guides/integrate/onboarding/end-users.mdx [hunk 4]
```diff
@@ -22,6 +22,34 @@ If you want to start automating the process of onboarding your users and let the
 
 ### Built-in register form
 
+#### Prompt registration form
+
+When users first access your application, they'll typically need to either log in or register.
+By default, redirecting from your application displays the login page.
+
+OpenID Connect (OIDC) allows you to control the initial screen by sending a [prompt](/docs/apis/openidoauth/endpoints#additional-parameters) in the authorization request.
```
Comment (mffap) [inline docs/docs/guides/integrate/onboarding/end-users.mdx:30]:
```suggestion
OpenID Connect (OIDC) allows you to control the initial screen by sending a [prompt](/docs/apis/openidoauth/endpoints#additional-parameters) parameter in the authorization request.
```

File: docs/docs/guides/integrate/onboarding/end-users.mdx [hunk 5]
```diff
@@ -22,6 +22,34 @@ If you want to start automating the process of onboarding your users and let the
...
+With the `prompt=create` the registration form/options will directly be shown to the user.
```
Comment (mffap) [inline docs/docs/guides/integrate/onboarding/end-users.mdx:31]:
```suggestion
With the `prompt=create`, the registration form/options will directly be shown to the user.
```

File: docs/docs/guides/integrate/onboarding/end-users.mdx [hunk 6]
```diff
@@ -22,6 +22,34 @@ If you want to start automating the process of onboarding your users and let the
...
+You can test the impact of the different prompts on your login UI in our [OIDC Playground](/docs/apis/openidoauth/authrequest).
+
+Unfortunately, SAML doesn't offer the same level of control over the initial screen.
+You won't be able to directly influence which page (login or registration) is shown through the SAML flow.
+
+#### Choose Registration Option
+
+If an organization does allow local user registration as well as registration with a social or enterprise login, the user will have to choose the option.
```
Comment (mffap) [inline docs/docs/guides/integrate/onboarding/end-users.mdx:39]:
```suggestion
If an organization allows local user registration as well as registration with a social or enterprise login, the user can choose an option to register.
```

**Review by mffap — CHANGES_REQUESTED** [review]
Should we, for completeness, include a short sentence about the organization scope to let users register to a pre-defined organization?

**Comment by hifabienne** [comment]
> Should we, for completeness, include a short sentence about the organization scope to let users register to a pre-defined organization?

added

**Review by mffap — APPROVED** [review]

**Pull request merged**